<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-KKIOJ2D" ignored="0" name="EdgeTest" tests="3" failures="0" timestamp="2021-02-17T23:21:27 ACDT" time="62.059" errors="0">
  <testcase name="@AfterClass closeApp" time="1.01" classname="com.tictoc.testcases.VerifySavingsDDT">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.common.TestReporting.ExtentReport.syncReportResult(ExtentReport.java:69)
at com.common.TestReporting.ExtentReport.flushReports(ExtentReport.java:61)
at com.tictoc.testcases.VerifySavingsDDT.closeApp(VerifySavingsDDT.java:77)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- @AfterClass closeApp -->
  <testcase name="VerifyRefiSavings" time="16.971" classname="com.tictoc.testcases.VerifySavingsDDT"/>
  <testcase name="VerifyRefiSavings" time="16.92" classname="com.tictoc.testcases.VerifySavingsDDT"/>
  <testcase name="VerifyRefiSavings" time="20.786" classname="com.tictoc.testcases.VerifySavingsDDT"/>
</testsuite> <!-- EdgeTest -->
